packages/plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -0,0 +1,147 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+ /* eslint camelcase: [0] */
+
+import '../..';
+import CiscoSpark from '@ciscospark/spark-core';
+import {assert} from '@ciscospark/test-helper-chai';
+import testUsers from '@ciscospark/test-helper-test-users';
+import uuid from 'uuid';
+
+describe(`plugin-calendar`, () => {
+  describe(`#meetings()`, () => {
+
+    let createdMeeting, mccoy, meetingParams, organizer, spark, spock;
+
+    before(`create test users`, () => testUsers.create({
+      count: 3,
+      config: {
+        entitlements: [
+          `spark`,
+          `squaredCallInitiation`,
+          `squaredInviter`,
+          `squaredRoomModeration`,
+          `webExSquared`,
+          `squaredFusionCal`
+        ]
+      }
+    })
+      .then((users) => {
+        [organizer, spock, mccoy] = users;
+
+        spark = new CiscoSpark({
+          credentials: {
+            authorization: organizer.token
+          },
+          config: {
+            device: {
+              preDiscoveryServices: {
+                whistlerServiceUrl: process.env.WHISTLER_API_SERVICE_URL || `http://internal-testing-services.wbx2.com:8084/api/v1`
+              }
+            }
+          }
+        });
+      }));
+
+    const hour = 1000 * 60 * 60;
+    const startInterval = new Date(new Date().getTime() + hour * 2).toISOString();
+    const meetingId = uuid.v4();
+
+    before(`creates a meeting`, () => {
+      meetingParams = {
```

Comment:
Also, is there a way to debug the predicates and transforms using karma? That'll gimme a better picture of what's happening.